pull_seqr_pedigrees.py

```diff
@@ -0,0 +1,57 @@
+import requests
+import getpass
+import argparse
+
+
+def pull_project_peds(args):
+    """
+    Uses the seqr API to retrieve a list of projects' pedigrees. Writes a single pedigree,
+    'pulled_seqr_pedigrees.txt', using the retrieved pedigrees. Failing projects are written
+    to 'projects_not_pulled.txt'
+    """
+    email=args.email
+    projects=args.projects
+
+    with requests.Session() as s:
+        p = s.post(
+            "https://seqr.broadinstitute.org/api/login", 
+            json={
+                'email': email,
+                'password': getpass.getpass(),
+            }
+        )
+        final_ped = open('seqr_pedigrees.txt', 'w')
```

Comment:
style comment - for consistency, you might want to use the 
with open("a.txt") as a, \
         open("b.txt") as b, ...:
syntax (https://legacy.python.org/dev/peps/pep-0008/#maximum-line-length)

Reply:
This  is a better approach, thanks!